governance-exercises/Poll Exercise 03.md

```diff
@@ -0,0 +1,110 @@
+# Poll Exercise 03
+
+---
+
+## Course of Action
+
+In the Exercise Prompt, I interpret "course of action" to encompass the immediate steps of responding to the Forum thread and other pre-drafting needs.
+
+As Governance Facilitator, my mandate includes:
+
+i) ensuring clarity and transparency around the process that led to the poll being submitted to governance; and
+
+ii) ensuring voters have access to sufficient information to make an informed vote.
+
+If polled in its current state, Emilia's proposal and Faraz's Reply on the Forum would fall short of both standards. Therefore, I would not run a governance poll. 
+
+*Note*: given the time constraints to complete this exercise, I will address at a high level my approach; briefly summarize my Forum responses; and defer the actual drafting of the Forum response. (I have confirmed this is ok with LFW.)
+
+---
+
+### Poll Timeline
+
+Emilia posted on Friday and stated "the poll should take place on Monday." Given the ambiguity in the Exercise prompt, I'll quickly review the various possibilities.
+
+#### *If Emilia means to request a standard weekly poll*   
+
+This is an unworkable turnaround time for a standard weekly poll. All needed information for a standard poll should be available by Thursday at the latest, if it is to run the following Monday. This gives the Governance Facilitator sufficient time to address all concerns, such as locating missing context; write the poll; and review and submit the poll on Friday. Rushing this process endangers both aspects of my mandate as a Governance Facilitator. (See [here](https://discord.com/channels/893112320329396265/899667119426371625/1111687437547212851).)
```

Comment:
Obviously I agree given you've quoted me saying the same. 

I will say unworkable is maybe more extreme than I'd describe it. It increases the chances of errors, and over the long term makes a notable contribution to facilitator burnout. But, for any individual case, it's not actually unworkable, just very frustrating. 

Agree that its a good reason to deny the poll this week in this case though. There is nothing that implies this specific poll needs to happen very quickly.